Android/app/src/main/java/io/github/project_travel_mate/utilities/DailyQuotesFragment.java

```diff
@@ -80,8 +80,8 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
         mHolder.rootLayout.setOnClickListener(view1 -> closeQuoteFragment());
         mHolder.share.setOnClickListener(view1 -> shareClicked());
         mHolder.negativeButton.setOnClickListener(view1 -> doNotShowClicked());
-
-        getQuote();
+        //TODO generating crash due to network calls on the main thread {use background treat }
```

Comment:
It isnt crashing. Revert this.

Reply:
This accurately  crash on my phone, that I why  I comment it to able to run my code